be/src/exec/vectorized/hash_join_node.cpp

```diff
@@ -56,11 +56,17 @@ Status HashJoinNode::init(const TPlanNode& tnode, RuntimeState* state) {
 
     const std::vector<TEqJoinCondition>& eq_join_conjuncts = tnode.hash_join_node.eq_join_conjuncts;
     for (const auto& eq_join_conjunct : eq_join_conjuncts) {
-        ExprContext* ctx = nullptr;
-        RETURN_IF_ERROR(Expr::create_expr_tree(_pool, eq_join_conjunct.left, &ctx));
-        _probe_expr_ctxs.push_back(ctx);
-        RETURN_IF_ERROR(Expr::create_expr_tree(_pool, eq_join_conjunct.right, &ctx));
-        _build_expr_ctxs.push_back(ctx);
+        ExprContext* left = nullptr;
```

Comment:
split to another pr?

Reply:
That's a dependency on the other PR. I would rebase after it merged.